README.md

```diff
@@ -1,7 +1,5 @@
 # Reliability Workbook
 
-[![Deploy to Azure](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Freliability-workbook%2Fmain%2Fworkbooks%2Fazuredeploy.json)
```

Comment:
Indeed, I was not aware that there may be environments where CloudShell is not available...

Reply:
Sure, we can target CloudShell with "Bash" option for this release, but that needs to be called out clearly in the documentation. Otherwise there are chances people might try with other environments.

Reply:
We need to document current assumptions and constraints.